static/gsApp/hooks/githubInstallationSelectInstall.tsx

```diff
@@ -0,0 +1,76 @@
+import {Fragment, useCallback, useRef} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/core/button';
+import type {SelectKey} from 'sentry/components/core/compactSelect';
+import GlobalModal from 'sentry/components/globalModal';
+import {IconLightning} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {openUpsellModal} from 'getsentry/actionCreators/modal';
+import withSubscription from 'getsentry/components/withSubscription';
+import type {Subscription} from 'getsentry/types';
+
+type Props = {
+  handleSubmit: (e: React.MouseEvent) => void;
+  has_scm_multi_org: boolean;
+  installationID: SelectKey;
+  isSaving: boolean;
+  subscription: Subscription;
+};
+
+function GithubInstallationSelectInstallButton({
+  has_scm_multi_org,
+  installationID,
+  subscription,
+  handleSubmit,
+  isSaving,
+}: Props) {
+  const organization = useOrganization();
+  const source = 'github.multi_org';
+  const mainContainerRef = useRef<HTMLDivElement>(null);
+  const handleModalClose = useCallback(() => {
+    mainContainerRef.current?.focus?.();
+  }, []);
+
+  if (installationID === '-1' || has_scm_multi_org) {
+    return (
+      <StyledButton onClick={handleSubmit} disabled={isSaving || !installationID}>
+        {t('Install')}
+      </StyledButton>
+    );
+  }
+
+  return (
+    <Fragment>
+      <GlobalModal onClose={handleModalClose} />
```

Comment:
Yoinked from the app startup as well: https://github.com/getsentry/sentry/blob/master/static/app/views/app/index.tsx#L259-L264